tests/functional/test_check.py

```diff
@@ -292,3 +292,22 @@ def test_check_include_work_dir_pkg(script):
     )
     assert matches_expected_lines(result.stdout, expected_lines)
     assert result.returncode == 1
+
+
+def test_check_integrity_errors_on_missing_files(data, script, tmpdir):
+    """
+    Work-in-progress failing test for a flag that detects broken packages
+    """
```

Comment:
The test shouldn't assume something failing. A docstring should refer to a passing test case.
```suggestion
    """Ensure that pip check detects a missing file post-install."""
```

Reply:
Ok, thanks!